spec/unit/classes/lvm_spec.rb

```diff
@@ -21,34 +23,70 @@
               'backup' => {
                 'size'              => '5G',
                 'mountpath'         => '/var/backups',
-                'mountpath_require' => true
+                'mountpath_require' => true,
+                'owner'             => 'foo',
+                'group'             => 'bar',
+                'mode'              => '0755'
               }
             }
           }
         }
       }
     end
 
-    it { should contain_physical_volume('/dev/sda2') }
-    it { should contain_physical_volume('/dev/sda3') }
-    it { should contain_volume_group('myvg').with({
-      :ensure           => 'present',
-      :physical_volumes => [ '/dev/sda2', '/dev/sda3', ]
-    }) }
-
-    it { should contain_logical_volume('opt').with( {
-      :volume_group => 'myvg',
-      :size         => '20G'
-    }) }
-    it { should contain_filesystem('/dev/myvg/opt') }
-    it { should contain_mount('/opt') }
-
-    it { should contain_logical_volume('backup').with({
-      :volume_group => 'myvg',
-      :size         => '5G'
-    }) }
-    it { should contain_filesystem('/dev/myvg/backup') }
-    it { should contain_mount('/var/backups') }
+    it {
```

Comment:
because nothing has changed from here down, it should follow the old formatting style.